.eslintignore

```diff
@@ -0,0 +1 @@
+lib/*.js
```

Comment:
Indeed, VS insiders edition didnt work that well in that regards...  Switching back